Mixin/UserInterface/Controllers/Home/Apps/Drag/HomeAppsFolderInteraction.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+protocol HomeAppsFolderInteraction {
+    
+    var dragInteraction: HomeAppsDragInteraction { get set }
+    var item: AppItem { get set }
+    var isDismissing: Bool { get set }
+    var wrapperView: UIView { get set }
+    
+}
+
+class HomeAppsFolderCreationInteraction: HomeAppsFolderInteraction {
```

Comment:
Variables should be declared as constant if possible. Same for `HomeAppsFolderDropInteraction` `HomeAppsOpenFolderInfo`